resources/event-sources/post-upgrade.sh

```diff
@@ -24,4 +24,10 @@ function delete_orphaned_ksvcs() {
   done
 }
 
+
+function delete_orphaned_policies() {
+    kubectl delete policy -n kyma-integration -l application-name
```

Comment:
I think we'd need `--ignore-not-found` to make it robust

Reply:
```suggestion
    kubectl delete policy --ignore-not-found -n kyma-integration -l application-name
```